### ISSUE TITLE
Bug: Play/Pause/Next/Previous/Stop buttons don't work

### ISSUE THREAD

**Issue by rusti4** [ModernFlyouts-Community/ModernFlyouts#532]
**Describe the bug:**
When the flyout appears with the above buttons, none of them work. Only the shuffle buttons work.

**OS Version:**
Edition	Windows 10 Pro
Version	21H1
Installed on	‎28/‎10/‎2020
OS build	19043.867
Experience	Windows Feature Experience Pack 120.2212.3030.0

**ModernFlyouts Version:**
0.9.1.0

**Comment by Samuel12321** [comment]
Which media app are you using

**Comment by de414** [comment]
I have a similar issue. Instead of none of the media buttons working on the UI itself, my keyboard control (only media controls, no issue with volume) stops working, only letting me control media through clicking on the UI.  I also have taskplay (lets you have skip, play, pause in taskbar) and that is no longer responsive either. I have a gmmk tkl if that helps any, my function row no longer works to skip songs and what not through Spotify.

**Comment by Cyberdroid1** [comment]
Hmmm, does it still happen if you quit ModernFlyouts?
Sounds like an app could be capturing all the keyboard shortcut inputs

**Comment by BobbyESP** [comment]
This happens to me too using Spotify.

**Comment by BobbyESP** [comment]
Ok, i see that the flyout gets opened, but it does nothing.

**Comment by svenjr** [comment]
I was having this same issue after initial install. A system restart fixed it for me

**Comment by BobbyESP** [comment]
Yep, same. I restarted the PC and now works

**Comment by de414** [comment]
Me too, now its not showing media interface through modern flyouts but that could've been a setting I changed trying to fix it. But my keyboard shortcuts work now so I'm happy lol

**Comment by rusti4** [comment]
Yes I have this issue with Spotify. My keyboard media keys work though.

**Comment by user3838** [comment]
I also have this issue. (with Spotify for example.)
The UI buttons are not working, grayed out.
When I restart ModernFlyouts the buttons start working.
When I then close Spotify and reopen it the buttons are again not working.
